servers/shareable-lists-api/src/admin/resolvers/index.ts

```diff
@@ -2,15 +2,15 @@ import { PocketDefaultScalars } from '@pocket-tools/apollo-utils';
 import { searchShareableList } from './queries/ShareableList';
 import { moderateShareableList } from './mutations/ShareableList';
 import { UserResolver } from '../../shared/resolvers/fields/User';
-import { PrismaBigIntResolver } from '../../shared/resolvers/fields/PrismaBigInt';
+import { ShareableListItem } from '../../database/types';
 
 export const resolvers = {
   ...PocketDefaultScalars,
   ShareableListComplete: {
     user: UserResolver,
   },
   ShareableListItem: {
-    itemId: PrismaBigIntResolver,
```

Comment:
Any idea why we needed this?

Reply:
I wonder if it was a schema change? I'll push to dev first